Deeploy/Targets/PULPOpen/TileConstraints/GEMMTileConstraint.py

```
@@ -262,23 +269,30 @@ def serializeTilingSolution(
+
         inputBaseOffsets, outputBaseOffsets = cls.extractBaseAddr(tilingSolution, targetMemLevel,
                                                                   operatorRepresentation, addrNames)
 
         transA = operatorRepresentation['transA']
         transB = operatorRepresentation['transB']
 
-        buffA = ctxt.lookup(operatorRepresentation['A'])
-        buffB = ctxt.lookup(operatorRepresentation['B'])
-        buffC = ctxt.lookup(operatorRepresentation['C'])
+        varA = operatorRepresentation['A']
+        varB = operatorRepresentation['B']
```

Ruff (0.14.5)

Local variable `varB` is assigned to but never used

Remove assignment to unused variable `varB`

(F841)